src/stylesheets/vendor/slick/slick-active.scss

```diff
@@ -0,0 +1,7 @@
+.slick-active .wheel-select__element {
+  cursor: pointer;
+}
+
+.slick-active .wheel-select__container {
```

Comment:
<!--
  0 failure: 
  1 warning:  Selector should b...
  
  
  DangerID: danger-id-Danger;
  File: src/stylesheets/vendor/slick/slick-active.scss;
  Line: 5;
-->

- :warning: Selector should be written in lowercase with hyphens (selector-class-pattern)